test failures are like
```
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Confluent.Kafka (>= 8.0.0) but Aspire.Confluent.Kafka 8.0.0 was not found. An approximate best match of Aspire.Confluent.Kafka 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Microsoft.Azure.Cosmos (>= 8.0.0) but Aspire.Microsoft.Azure.Cosmos 8.0.0 was not found. An approximate best match of Aspire.Microsoft.Azure.Cosmos 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Microsoft.Data.SqlClient (>= 8.0.0) but Aspire.Microsoft.Data.SqlClient 8.0.0 was not found. An approximate best match of Aspire.Microsoft.Data.SqlClient 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.MongoDB.Driver (>= 8.0.0) but Aspire.MongoDB.Driver 8.0.0 was not found. An approximate best match of Aspire.MongoDB.Driver 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.MySqlConnector (>= 8.0.0) but Aspire.MySqlConnector 8.0.0 was not found. An approximate best match of Aspire.MySqlConnector 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Npgsql (>= 8.0.0) but Aspire.Npgsql 8.0.0 was not found. An approximate best match of Aspire.Npgsql 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Npgsql.EntityFrameworkCore.PostgreSQL (>= 8.0.0) but Aspire.Npgsql.EntityFrameworkCore.PostgreSQL 8.0.0 was not found. An approximate best match of Aspire.Npgsql.EntityFrameworkCore.PostgreSQL 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Oracle.EntityFrameworkCore (>= 8.0.0) but Aspire.Oracle.EntityFrameworkCore 8.0.0 was not found. An approximate best match of Aspire.Oracle.EntityFrameworkCore 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.Pomelo.EntityFrameworkCore.MySql (>= 8.0.0) but Aspire.Pomelo.EntityFrameworkCore.MySql 8.0.0 was not found. An approximate best match of Aspire.Pomelo.EntityFrameworkCore.MySql 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.RabbitMQ.Client (>= 8.0.0) but Aspire.RabbitMQ.Client 8.0.0 was not found. An approximate best match of Aspire.RabbitMQ.Client 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj : error NU1603: Warning As Error: TestProject.IntegrationServiceA depends on Aspire.StackExchange.Redis (>= 8.0.0) but Aspire.StackExchange.Redis 8.0.0 was not found. An approximate best match of Aspire.StackExchange.Redis 8.0.1 was resolved. [/datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj]
[restore]   Failed to restore /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj (in 24.87 sec).
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.AppHost (>= 8.0.0) but Aspire.Hosting.AppHost 8.0.0 was not found. An approximate best match of Aspire.Hosting.AppHost 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.Azure (>= 8.0.0) but Aspire.Hosting.Azure 8.0.0 was not found. An approximate best match of Aspire.Hosting.Azure 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.Azure.CosmosDB (>= 8.0.0) but Aspire.Hosting.Azure.CosmosDB 8.0.0 was not found. An approximate best match of Aspire.Hosting.Azure.CosmosDB 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.Kafka (>= 8.0.0) but Aspire.Hosting.Kafka 8.0.0 was not found. An approximate best match of Aspire.Hosting.Kafka 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.MongoDB (>= 8.0.0) but Aspire.Hosting.MongoDB 8.0.0 was not found. An approximate best match of Aspire.Hosting.MongoDB 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.MySql (>= 8.0.0) but Aspire.Hosting.MySql 8.0.0 was not found. An approximate best match of Aspire.Hosting.MySql 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.NodeJs (>= 8.0.0) but Aspire.Hosting.NodeJs 8.0.0 was not found. An approximate best match of Aspire.Hosting.NodeJs 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.Oracle (>= 8.0.0) but Aspire.Hosting.Oracle 8.0.0 was not found. An approximate best match of Aspire.Hosting.Oracle 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.PostgreSQL (>= 8.0.0) but Aspire.Hosting.PostgreSQL 8.0.0 was not found. An approximate best match of Aspire.Hosting.PostgreSQL 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.RabbitMQ (>= 8.0.0) but Aspire.Hosting.RabbitMQ 8.0.0 was not found. An approximate best match of Aspire.Hosting.RabbitMQ 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.Redis (>= 8.0.0) but Aspire.Hosting.Redis 8.0.0 was not found. An approximate best match of Aspire.Hosting.Redis 8.0.1 was resolved.
[restore] /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj : error NU1603: Warning As Error: TestProject.AppHost depends on Aspire.Hosting.SqlServer (>= 8.0.0) but Aspire.Hosting.SqlServer 8.0.0 was not found. An approximate best match of Aspire.Hosting.SqlServer 8.0.1 was resolved.
[restore]   Failed to restore /datadisks/disk1/work/B74F09FD/w/A52B093F/e/testassets/testproject/TestProject.AppHost/TestProject.AppHost.csproj (in 20.84 sec).


```

--- a/tests/testproject/Directory.Packages.Helix.props
+++ b/tests/testproject/Directory.Packages.Helix.props
@@ -8,7 +8,7 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
     <_PackageVersion>$(PackageVersion)</_PackageVersion>
     <!-- If package versions are being stabilized, the version used shouldn't have the prerelease tag in it. -->
-    <_PackageVersion Condition="'$(StabilizePackageVersion)' == 'true'">8.0.0</_PackageVersion>
+    <_PackageVersion Condition="'$(StabilizePackageVersion)' == 'true'">8.0.1</_PackageVersion>
   </PropertyGroup>
 
   <ItemGroup>

